src/composables/trade/useGnosis.ts

```diff
@@ -23,7 +23,17 @@ import useNumbers from '../useNumbers';
 
 // TODO: get correct app id
 const GNOSIS_APP_ID = 2;
-const appData = '0x' + GNOSIS_APP_ID.toString(16).padStart(64, '0');
+const APP_DATA = '0x' + GNOSIS_APP_ID.toString(16).padStart(64, '0');
+const HIGH_FEE_THRESHOLD = 0.2;
+const INITIAL_STATE = {
+  errors: {
+    feeExceedsPrice: false,
+    priceExceedsBalance: false
+  },
+  warnings: {
+    highFees: false
+  }
+};
```

Comment:
Just define initial state outside the composable function and then use it internally.
```suggestion
const state = reactive({
  errors: {
    feeExceedsPrice: false,
    priceExceedsBalance: false
  },
  warnings: {
    highFees: false
  }
});
```
Then in the composable below you can update like:
```js
state.errors.feeExceedsPrice = true;
```
Then to export:
```js
return {
  ...toRefs(state),
}
```